2025-04-24 21:15:21,598 INFO    MainThread:19862 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/ace/miniconda3/envs/lnenv/lib/python3.8/site-packages/gym/envs/user/protect/wandb/run-20250424_211521-2_target/logs/debug.log
2025-04-24 21:15:21,598 INFO    MainThread:19862 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/ace/miniconda3/envs/lnenv/lib/python3.8/site-packages/gym/envs/user/protect/wandb/run-20250424_211521-2_target/logs/debug-internal.log
2025-04-24 21:15:21,598 INFO    MainThread:19862 [wandb_init.py:init():761] calling init triggers
2025-04-24 21:15:21,598 INFO    MainThread:19862 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-04-24 21:15:21,598 INFO    MainThread:19862 [wandb_init.py:init():784] starting backend
2025-04-24 21:15:21,598 INFO    MainThread:19862 [wandb_init.py:init():788] sending inform_init request
2025-04-24 21:15:21,600 INFO    MainThread:19862 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-24 21:15:21,600 INFO    MainThread:19862 [wandb_init.py:init():798] backend started and connected
2025-04-24 21:15:21,606 INFO    MainThread:19862 [wandb_init.py:init():891] updated telemetry
2025-04-24 21:15:21,620 INFO    MainThread:19862 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-04-24 21:15:24,440 INFO    MainThread:19862 [wandb_init.py:init():990] starting run threads in backend
2025-04-24 21:15:24,537 INFO    MainThread:19862 [wandb_run.py:_console_start():2375] atexit reg
2025-04-24 21:15:24,538 INFO    MainThread:19862 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-04-24 21:15:24,538 INFO    MainThread:19862 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-04-24 21:15:24,538 INFO    MainThread:19862 [wandb_run.py:_redirect():2315] Redirects installed.
2025-04-24 21:15:24,539 INFO    MainThread:19862 [wandb_init.py:init():1032] run started, returning control to user process
2025-04-24 21:16:58,565 INFO    MainThread:19862 [wandb_run.py:_finish():2112] finishing run huhugiao-bit/protect_vs_invade/2_target
2025-04-24 21:16:58,565 INFO    MainThread:19862 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 0
2025-04-24 21:16:58,566 INFO    MainThread:19862 [wandb_run.py:_restore():2322] restore
2025-04-24 21:16:58,566 INFO    MainThread:19862 [wandb_run.py:_restore():2328] restore done
